specification/apicenter/data-plane/ApiCenter.DataApi/preview/2024-02-01-preview/azure.apicenter.dataapi.json

```diff
@@ -0,0 +1,1644 @@
+          }
+        ]
+      }
+    },
+    "Models.Api": {
+      "type": "object",
+      "description": "API resource model.",
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "The resource name.",
+          "maxLength": 50,
+          "readOnly": true
+        },
+        "title": {
+          "type": "string",
+          "description": "The name of the API.",
+          "maxLength": 50
+        },
+        "summary": {
+          "type": "string",
+          "description": "Short description of the API.",
+          "maxLength": 200
+        },
+        "description": {
+          "type": "string",
+          "description": "The description of the API",
+          "maxLength": 1000
+        },
+        "kind": {
+          "$ref": "#/definitions/Models.ApiKind",
+          "description": "Kind of API. For example, REST or GraphQL."
+        },
+        "lifecycleStage": {
+          "$ref": "#/definitions/Models.ApiLifecycleStage",
+          "description": "Current lifecycle stage of the API."
+        },
+        "termsOfService": {
+          "$ref": "#/definitions/Models.ApiTermsOfService",
+          "description": "Terms of service for the API."
+        },
+        "license": {
+          "$ref": "#/definitions/Models.ApiLicenseModel",
+          "description": "The license information for the API."
+        },
+        "externalDocumentation": {
+          "type": "array",
+          "description": "External documentation",
+          "items": {
+            "$ref": "#/definitions/Models.ApiExternalDocumentation"
+          },
+          "x-ms-identifiers": []
+        },
+        "contacts": {
+          "type": "array",
+          "description": "Points of contact for the API.",
+          "items": {
+            "$ref": "#/definitions/Models.ApiContact"
+          },
+          "x-ms-identifiers": []
+        },
+        "customProperties": {
+          "type": "object",
+          "description": "The custom metadata defined for API entries.",
+          "additionalProperties": {
+            "type": "string"
+          }
+        },
+        "lastUpdated": {
+          "type": "string",
+          "format": "date-time",
+          "description": "Last updated date and time."
+        }
+      },
+      "required": [
+        "name",
+        "title",
+        "kind",
+        "lastUpdated"
+      ]
+    },
+    "Models.ApiContact": {
+      "type": "object",
+      "description": "API contact information",
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "Name of the contact.",
+          "maxLength": 100
+        },
+        "url": {
+          "type": "string",
+          "format": "uri",
+          "description": "URL for the contact."
+        },
+        "email": {
+          "type": "string",
+          "description": "Email address for the contact.",
+          "maxLength": 100
+        }
+      },
+      "required": [
```

Comment:
Only `name` can be required as required in `ApiContact` entity (which btw should be just `Contact`, because it might be re-used elsewhere).